workflow/usage.rst

```diff
@@ -265,8 +266,8 @@ order:
     * ``workflow.[workflow name].enter.[place name]``
 
 ``workflow.entered``
-    Similar to ``workflow.enter``, except the marking store is updated before this
-    event (making it a good place to flush data in Doctrine).
+    The object has entered a state and the marker store is updated (making it a good
```

Comment:
* has entered in the place (I'm not so fluent, but we should talk about place and not state)
* `and the marking store`

Reply:
Fixed!